minidump-common/src/format.rs

```diff
@@ -806,8 +810,8 @@ pub struct CONTEXT_AMD64 {
     /// This is defined as a union in the C headers, but also
     /// ` MAXIMUM_SUPPORTED_EXTENSION` is defined as 512 bytes.
     ///
-    /// Callers that want to access the underlying data can use `Pread` to read either
-    /// an `XMM_SAVE_AREA32` or `SSE_REGISTERS` struct from this raw data as appropriate.
+    /// Callers that want to access the underlying data can use [`Pread`] to read either
```

Comment:
Does this link to `Pread` work given that it comes from the `scroll` crate?

Reply:
Yes, that links correctly to other crates (as long as you don't build with `--no-deps`)